pkg/resolver/searchComplete_test.go

```diff
@@ -238,3 +236,53 @@ func Test_SearchCompleteWithLabel_Query(t *testing.T) {
 	// Verify response
 	AssertStringArrayEqual(t, result, expectedProps, "Error in Test_SearchCompleteWithLabel_Query")
 }
+
```

Comment:
Do we have a test that covers `label`?

Reply:
yes, label is covered in `Test_SearchCompleteWithLabel_Query()`

Reply:
Can you add a test case for general search too? This one is SearchComplete.

Reply:
I see an issue with image `quay.io/stolostron/search-v2-api:2.7.0-PR91-4886e6c66e49108ff1bfb05e1ca4bfb1cb8e22c4` on my cluster
Step 1: 
Search with label.
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/32376120/194360702-12732896-2f2b-48bb-aed3-ebb0d74d439c.png">

Step 2:
Add `cluster` filter
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/32376120/194361171-4f43dff3-fa7c-4d8b-955c-37365bad5620.png">